src/plugins/checkout-utils-mixin.js

```diff
@@ -114,36 +114,42 @@ export default {
 		/* Handle Various errors from GraphQL
 		 * @param {Object} errorResponse contains errors node with array of errors
 		 */
-		showCheckoutError(errorResponse, ignoreAuth = false) {
+		showCheckoutError(errorResponse = [], ignoreAuth = false) {
 			checkInjections(this, injections);
 
 			// const errors = _get(errorResponse, 'errors');
 			let errorMessages = '';
 			// When validation or checkout fails and errors object is returned along with the data
-			errorResponse.forEach(({ error, value }) => {
-				let errorMessage = value;
+			errorResponse.forEach(({
+				error,
+				value,
+				extension,
+				message
+			}) => {
+				let errorMessage = value || message;
+				const errorType = error || extension?.code;
 
 				/* eslint-disable max-len */
 				// update error messages for new checkout context (Original messages reference a different user flow)
-				if (error === 'ERROR_OWN_LOAN') {
+				if (errorType === 'ERROR_OWN_LOAN') {
 					// Original Message: As a Kiva borrower, you cannot support your own fundraising loan on Kiva.  Please go back to the basket to remove your loan.
 					errorMessage = 'As a Kiva borrower, you cannot support your own fundraising loan on Kiva. Please remove your loan before completing checkout.';
 				}
-				if (error === 'ERROR_OVER_DAILY_LIMIT') {
+				if (errorType === 'ERROR_OVER_DAILY_LIMIT') {
```

Comment:
I was able to verify existing signature works with this error and that the new signature works with the UPC error.